lib/chainsync/pyproject.toml

```diff
@@ -16,14 +16,14 @@ classifiers = [
     "Operating System :: OS Independent",
 ]
 
-dependencies = [
-    # Lateral dependencies across subpackages are pointing to github
-    # TODO test that outermost pip install overwrites this
-    # TODO move this to either this monorepo or seperate repo
-    #"fixedpointmath @ git+https://github.com/delvtech/agent_0.git/#subdirectory=lib/fixedpointmath",
-    # TODO this might not work since it's not on github yet, need to test
-    #"elfpy @ git+https://github.com/delvtech/elf-simulations.git/#subdirectory=lib/elfpy",
-    #"ethpy @ git+https://github.com/delvtech/elf-simulations.git/#subdirectory=lib/ethpy",
+[project.optional-dependencies]
+# This flag installs all dependencies and should be ran when installing this subpackage in isolation
+with-dependencies = [
+    "chainsync[base, lateral]"
+]
+base = [
+    # TODO move fixedpointmath to lateral if this package comes back to this monorepo
```

Comment:
I think we decided to move it to its own repo instead of bringing it back here. This monorepo is for packages that are still under development.

Reply:
Sounds good. `fixedpointmath` should belong here then.